src/hamster/overview.py

```diff
@@ -538,6 +542,21 @@ def on_timeout(self):
         # The timeout will stop if returning False
         return True
 
+    def on_help_clicked(self, menu):
+        uri = "help:hamster-time-tracker"
+        try:
+            gtk.show_uri(gdk.Screen().get_default(), uri,
+                         gdk.CURRENT_TIME)
+        except gi.repository.GLib.Error:
+            msg = sys.exc_info()[1].args[0]
+            dialog = gtk.MessageDialog(self.window, 0, gtk.MessageType.ERROR,
+                                       gtk.ButtonsType.CLOSE,
+                                       _("Failed to open {}").format(uri))
+            fmt = _('Error: "{}" - is a help browser installed on this Computer?')
```

Comment:
```suggestion
            fmt = _('Error: "{}" - is a help browser such as yelp installed on this Computer?')
```
Mentioning `yelp` to further guide the user ?

Reply:
I don't think `yelp` is the only possibility. The KDE Help browser is another I guess. I wouldn't want to be too specific, or biased towards a certain environment. Next year "yelp" may have been replaced by something else in GNOME.

Reply:
Moreover, the name "yelp" is also pretty much hidden in GNOME environments. I believe that using such specific names is pretty much deprecated in GUI environments. "Help browser" should mean something to most users, "yelp" definitely doesn't. It's true that users need to figure out which package they need to install to get the help browser - but honestly that's GNOMEs or distributor's job, not ours.

Reply:
That makes sense. We'll just add a line in the [dependencies section](https://github.com/projecthamster/hamster#dependencies).

Reply:
Make sure to make it a "weak" dependency though. In distro speak, it would be a "Recommends:", not a "Requires:".

Reply:
The KDE help browser `khelpcenter` does not seem to read these `mallard` pages.
(they do not show up when searching for "hamster").
Yelp seems to be the only solution currently. No big deal; it is both nice and lightweight.
https://github.com/mwilck/hamster/pull/3 starts with a proposition about that.